constellation-app/constellation
# Fix null pointer and other exceptions in schema view

### Prerequisites


    for the "CI / Unit Tests") to complete first. Failed Unit tests can be 
    debugged by adding the label "verbose logging" to the GitHub PR.

### Description of the Change


Added synchronize to functions so that rapid closing and opening of schema view doesn't throw errors.

### Alternate Designs


N/A

### Why Should This Be In Core?


N/A

### Benefits


Bug fixes

### Possible Drawbacks


None

### Verification Process



1. Open consty
2. Open a graph
3. Rapidly open and close schema view (schema view can be opened with ctrl-shift-s)
4. Observe no exceptions are thrown


### Applicable Issues


https://github.com/constellation-app/constellation/issues/2144
